packages/libs/chainwebjs/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./node_modules/@kadena-dev/heft-rig/profiles/default/tsconfig-base.json",
+  "compilerOptions": {
+    "types": ["heft-jest", "node"],
```

Comment:
Can we get rid of "node". The lib should work in a browser environment as well. Is that possible? See also #47